src/Microsoft.Diagnostics.EventFlow.Core/Microsoft.Diagnostics.EventFlow.Core.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <Description>Defines core interfaces and types that comprise Microsoft.Diagnostics.EventFlow library.</Description>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
-    <VersionPrefix>1.4.1</VersionPrefix>
+    <VersionPrefix>1.4.2</VersionPrefix>
```

Comment:
I made a few changes recently--please bump up to 1.4.3

Reply:
If you don't mind, I would appreciate you making the changes on merge.

I need to add usage info to the readme. Is it okay to do that in another PR?

Thanks for the feedback.

Reply:
Sure, that will be fine